### PR TITLE
send user when sendBeacon is not supported

### PR DESCRIPTION
**Pull request by RobertoUa**
`withCredentials` wouldn't work.

Also it's probably not the best way to use regex for cookies :/

### CODE REVIEW

**Comment by RobertoUa** [comment]
@shamansir

File: src/anm/analytics.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ var Analytics = function () {
             if (navigator.sendBeacon) {
                 navigator.sendBeacon(animatronUrl, data);
             } else {
-                engine.ajax(animatronUrl, null, null, 'POST', null, data, false);
+                var auth = document.cookie.match(/_animatronauth=(\w+);/)[1];
```
Comment (RobertoUa) [inline src/anm/analytics.js]: maybe I should check for null and length > 0 before using `[1]`

Comment (shamansir) [inline src/anm/analytics.js]: Yes, you should :). Also, final cookie may have no `;` sign.

**Comment by RobertoUa** [comment]
updated

**Pull request merged**